src/main/java/omero/gateway/Gateway.java

```diff
@@ -452,6 +452,24 @@ public String getSessionId(ExperimenterData user)
         return null;
     }
 
+    /**
+     * Return the host
+     *
+     * @param user The user to get the host for
+     * @return See above
+     * @throws DSOutOfServiceException
+     *             If the connection is broken, or not logged in
+     */
+    public String getHost(ExperimenterData user)
+            throws DSOutOfServiceException {
+        Connector c = getConnector(new SecurityContext(user.getGroupId()),
```

Comment:
Why is the Connector needed in that case, just to check if the user is logged in?

Reply:
just to check if we have a connection